src/base/net/downloadmanager.cpp

```diff
@@ -123,8 +123,10 @@ namespace
 
         // Spoof HTTP Referer to allow adding torrent link from Torcache/KickAssTorrents
         request.setRawHeader("Referer", request.url().toEncoded().data());
-        // Accept gzip
+#ifdef QT_NO_COMPRESS
+        // Manually decompress with gzip
```

Comment:
Why "Manually decompress"? IIRC, the affected code is to say server that we accept gzipped data so it is independent from whether it will be decompressed manually or by Qt.

Reply:
Anyway your comment is confusing. It should say that we **can** decompress gzip "manually" so we still accept it.